src/reducers/resourceReducer.js

```diff
@@ -43,6 +46,23 @@ export function filteredResource(state = initialState.filteredResource, action)
 
 export function savedResource(state = initialState.savedResource, action) {
     switch (action.type) {
+        case types.LOAD_RESOURCE_DATA_SUCCESS:
+          const query = qs.parse(window.location.search.replace('?', ''));
+          let selectedResourceIds = [];
+          if (query.resources) {
+            selectedResourceIds = query.resources.split(',');
+          }
+
+          const selectedResources = [];
+          selectedResourceIds.forEach(selectedResourceId => {
+            action.resource.forEach(resource => {
+              if (resource.id === selectedResourceId) {
+                selectedResources.push(resource);
+              }
+            });
+          })
+
+          return Object.assign([], state, selectedResources);
```

Comment:
Does it cause any issue if this ends if `selectedResources` turns out to be undefined ?